src/convertSelectorsToCSS.ts

```diff
@@ -0,0 +1,9 @@
+function convertSelectorsToCSS(selectors: string[]): string {
```

Comment:
as this is only used in one place does it really need to be in it's own file?

Reply:
Oops! I had merged seconds before you left this comment ha! Me personally I enjoy keeping small isolated modules/files but that's just a preference, would you rather it not be in it's own file?